docs/advanced/lockfile_diffs.md

```diff
@@ -0,0 +1,77 @@
+# Calculate Lockfile diffs
+
+![pixi-diff demo](https://raw.githubusercontent.com/pavelzw/pixi-diff/refs/heads/main/.github/assets/demo/demo-light.gif#only-light)
+![pixi-diff demo](https://raw.githubusercontent.com/pavelzw/pixi-diff/refs/heads/main/.github/assets/demo/demo-dark.gif#only-dark)
+
+It can happen that you want to know what changed in your lockfile after repeatedly adding and removing dependencies within a pull request.
+For this, you can use [pavelzw/pixi-diff](https://github.com/pavelzw/pixi-diff) to calculate the differences between two lockfiles.
+This can be leveraged in combination with [pavelzw/pixi-diff-to-markdown](https://github.com/pavelzw/pixi-diff-to-markdown) to generate a markdown file that shows the diff in a human-readable format.
+With [henriklovhaug/md-tui](https://github.com/henriklovhaug/md-tui), you can even render the markdown file in the terminal.
+
+!!!tip "Install the tools globally"
+    All of the above-mentioned tools are available on conda-forge and can be installed using [`pixi global install`](../features/global_tools.md).
+
+    ```bash
+    pixi global install --environment pixi-diff pixi-diff pixi-diff-to-markdown
```

Comment:
Any reason why you'd install those tools in the same environment?

Reply:
they semantically belong together but we don't really get any benefit from shared packages or anything since one is rust the other python, i can also put them in different envs if you like

Reply:
Yes, let's change those two instructions to the following then:

```bash
pixi global install pixi-diff pixi-diff-to-markdown md-tui
```

Reply:
```suggestion
    pixi global install pixi-diff
    pixi global install pixi-diff-to-markdown
```